python/vast-python.spdx

```diff
@@ -0,0 +1,631 @@
+PackageName: antlr4-python3-runtime
```

Comment:
Can you append this file to VAST.spdx instead of it being its own package (with some delimiters making it clear where Python begins and ends), or change the installation logic in CMake to also install this SBOM file?